source/Ship.cpp

```diff
@@ -3533,6 +3534,13 @@ double Ship::TurnRate() const
 
 
 
+double Ship::TrueTurnRate() const
+{
+	return attributes.Get("turn") / InertialMass() * 1. / (1. + slowness * .05);
```

Comment:
As best I can tell, TurnRate() just gives the ship's turning, without taking slowing into account. Part of the reason for including it is so that we can see what effect slowing weapons are actually having.

Reply:
```suggestion
	return TurnRate() * 1. / (1. + slowness * .05);
```

Reply:
Ah, thank you. I thought Warpcore was suggestion replacing the whole line with that call, instead of just that part.

Reply:
np